content/blogposts/2024/cody-vscode-1-14-0-release.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Cody for VS Code v1.14.0: Now with bigger context windows and a refreshed chat UI"
+authors:
+  - name: Alex Isken
+    url: https://handbook.sourcegraph.com/team/#alex-isken
+  - name: Justin Dorfman
+    url: https://handbook.sourcegraph.com/team/#justin-dorfman
+publishDate: 2024-04-17T10:00-01:00
+description: "Cody for VS Code v1.14.0 is now available. This update includes expanded input and output context windows, a new chat interface, improved unit tests, and more."
+tags: [blog]
+slug: "cody-vscode-1-14-0-release"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1-14-release/cody-vscode-1.14.0-og-image.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1-14-release/cody-vscode-1.14.0-og-image.png
+--- 
+
+[Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v1.14.0 is now available! This update includes context window expansions, a new chat interface, easier ways to add context to chat, new models, and more.
+
+## Bigger and better context windows to improve chat
+
+We’ve historically limited Cody to a maximum context window of 7,000 tokens, meaning that Cody won’t pass more than 7,000 tokens (small chunks of text) to the underlying LLMs. This is mostly to make sure that _too much_ context isn’t passed into the LLM, which can lead to poor context recall and then poor answer quality. 
+
+Because of Claude 3’s [improved context recall](https://www.google.com/url?q=https://www.anthropic.com/news/claude-3-family%23:~:text%3DLong%2520context%2520and%2520near%252Dperfect%2520recall&sa=D&source=docs&ust=1713389874874139&usg=AOvVaw2xo3VN2mbBXrMP3p9cCkLc), we can now expand that context window without degrading chat quality. We’ve made huge updates to Cody’s maximum context window for Claude 3 Sonnet and Opus models:
+
+* 30,000 tokens of user-defined context (user text plus @-mentioned files)
```

Comment:
I usually wrap `@mention-s` in backticks

Reply:
Resolved in new commit